modules/multipath/data/org.freedesktop.UDisks2.Multipath.xml

```diff
@@ -0,0 +1,135 @@
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General
+ Public License along with this library; if not, write to the
+ Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+ Boston, MA 02111-1307, USA.
+-->
+
+  <!-- ******************************************************************* -->
+
+  <!--
+    org.freedesktop.UDisks2.Drive.Multipath:
+    @short_description: Multipath interface exported on disk objects
+  -->
+  <interface name="org.freedesktop.UDisks2.Drive.Multipath">
+    <!-- Multipath:
+        If the disk drive is multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
+    <property name="Multipath" type="o" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Block.Multipath:
+    @short_description: Multipath interface exported on block objects
+  -->
+  <interface name="org.freedesktop.UDisks2.Block.Multipath">
+    <!-- Path:
+        If the block is a multipath device slave, this is set to the
+        org.freedesktop.UDisks2.Multipath.Path object that it
+        corresponds to. It is '/' otherwise.
+    -->
+    <property name="Path" type="o" access="read"/>
+    <!-- Multipath:
+        If block is(or part of) multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
+    <property name="Multipath" type="o" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Multipath:
+    @short_description: A multipath device.
+  -->
+  <interface name="org.freedesktop.UDisks2.Multipath">
+    <!-- WWID:
+        The WWID of certain multipath device(mpath).
+        It depends on "getuid_callout" configuration of multipathd.
+        Normally, it's SCSI VPD page 0x83 device ID or SCSI VPD page
+        0x80(rare) device ID of the disk device.
+    -->
+    <property name="WWID" type="s" access="read"/>
+    <!-- Name:
+        The alias of certain multipath device. For example: "mpatha".
+    -->
+    <property name="Name" type="s" access="read"/>
+    <!-- Block:
+        This is set to the org.freedesktop.UDisks2.Block object of
+        the /dev/dm-XX block
+    -->
+    <property name="Block" type="o" access="read"/>
+    <!-- Drive:
+        This is set to the org.freedesktop.UDisks2.Drive object.
+    -->
+    <property name="Drive" type="o" access="read"/>
+    <!-- Paths:
+        Array of org.freedesktop.UDisks2.Multipath.Path object paths.
+    -->
+    <property name="Paths" type="ao" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Multipath.Path:
+    @short_description: A path of multipath path group device.
+  -->
+  <interface name="org.freedesktop.UDisks2.Multipath.Path">
```

Comment:
Although I can imagine a user eventually wanting this information, it might be a bit more detail than is warranted in the initial version since we don't know of any potential users at this point. My initial thought is: YAGNI (But I realize this is a slippery slope when building an API without any specific user in waiting).